imperial_coldfront_plugin/settings.py

```diff
@@ -0,0 +1,6 @@
+"""Settings for the Imperial Coldfront plugin.
+
+These are imported into the project level settings by the Coldfront plugin mechanism.
+"""
+
+INVITATION_TOKEN_TIMEOUT = 86400
```

Comment:
Does that mean the invite token will timeout after 24 hours?

Reply:
It does. But actually it should probably be longer and we can make that clearer.

Reply:
Ok, so what if the invite is sent at the end of a Friday, then the 24 hours are not very helpful - what is usually a minimal good number of hours for the timeout?